static/app/views/performance/charts/chart.tsx

```diff
@@ -24,6 +25,7 @@ type Props = {
   disableXAxis?: boolean;
   chartColors?: string[];
   loading: boolean;
+  isLineChart?: boolean;
```

Comment:
Are line charts and area charts the only charts we'll support here? If there will be additional chart types, perhaps something like
```typescript
chartType?: 'area' | 'line' | ...
```
may be more suitable

Reply:
This file might go away entirely, which is why I wasn't too worried about using a one off prop like this. Good catch though.